libs/application/templates/social-insurance-administration/medical-and-rehabilitation-payments/src/forms/Prerequisites/externalDataSubSection.ts

```diff
@@ -40,39 +40,50 @@ export const externalDataSubSection = buildSubSection({
       }),
       dataProviders: [
         buildDataProviderItem({
-          // TODO: Update text!
           provider: NationalRegistryUserApi,
-          title: socialInsuranceAdministrationMessage.pre.skraInformationTitle,
-          subTitle: '',
+          title:
+            medicalAndRehabilitationPaymentsFormMessage.pre
+              .skraInformationTitle,
+          subTitle:
+            medicalAndRehabilitationPaymentsFormMessage.pre
+              .skraInformationDescription,
         }),
         buildDataProviderItem({
-          // TODO: Update text?
           provider: NationalRegistrySpouseApi,
           title: '',
         }),
         buildDataProviderItem({
           provider: UserProfileApi,
-          title: socialInsuranceAdministrationMessage.pre.contactInfoTitle,
+          title:
+            medicalAndRehabilitationPaymentsFormMessage.pre.contactInfoTitle,
           subTitle:
-            socialInsuranceAdministrationMessage.pre.contactInfoDescription,
+            medicalAndRehabilitationPaymentsFormMessage.pre
+              .contactInfoDescription,
         }),
```

Comment:
Add `SocialInsuranceAdministrationApplicantApi` because we are getting the bank account from TR
![image](https://github.com/user-attachments/assets/27eb959b-b9fa-474c-b436-279677756e4c)

```suggestion
        }),
        buildDataProviderItem({
          provider: SocialInsuranceAdministrationApplicantApi,
          title:
            socialInsuranceAdministrationMessage.pre
              .socialInsuranceAdministrationTitle,
          subTitle:
            socialInsuranceAdministrationMessage.pre
              .socialInsuranceAdministrationDescription,
        }),
```

Reply:
New provider added